sdk/textanalytics/Azure.AI.TextAnalytics/tests/AnalyzeOperationMockTests.cs

```diff
@@ -707,6 +707,40 @@ public void AnalyzeOperationAnalyzeSentimentWithTwoActions()
 
         #endregion Analyze sentiment
 
+        #region Extract summary
+
+        [Test]
+        public void AnalyzeOperationExtractsSummaryWithTwoActions()
```

Comment:
Can we add a test for a single action case with a long document? e.g. JS uses: https://github.com/Azure/azure-sdk-for-js/blob/7e1680e3f8a82e5f41c1c8ccdfa433f9f2c250c0/sdk/textanalytics/ai-text-analytics/test/public/textAnalyticsClient.spec.ts#L930

Reply:
This would be a live test, so I can only add it when the output side is implemented. Do you have tests for long documents in all endpoints in JS or this is something added only for extractive text summarization?

Reply:
>  I can only add it when the output side is implemented.

To clarify, you're saying processing the response received from the service is not yet implement in the SDK? that is fine if so.

The nature of the input document depends on the action you want to apply to it so in this case I think a long document is needed to make sure we get a useful summary.

Reply:
> To clarify, you're saying processing the response received from the service is not yet implement in the SDK?

Correct. We just have the "sending" part implemented in this PR.